Tasks/Safina/Task1/Add Task1.py

```diff
@@ -0,0 +1,8 @@
+import math
+a = 20000
+b = 5
+c = 15
```

Comment:
By adapting these values for the int type and using them in the math expression you get the float number as answer, which tends to accumulate some error, it's better to use decimal for money-related operations, -10 points for the logical mistake